src/code/home/animated_tiles.asm

```diff
@@ -242,20 +242,20 @@ LoadAnimatedTilesFrameAtOffset::
 
 AnimateTideTilesGroup::
     ld   h, HIGH(AnimatedTiles) + $1              ; $1C16: $26 $6B
-    jr   AnimateTilesSlowSpeed                    ; $1C18: $18 $0A
+    jr   AnimateTilesSlowVelocity                 ; $1C18: $18 $0A
 
 AnimateVillageTilesGroup::
     ld   h, HIGH(AnimatedTiles) + $2              ; $1C1A: $26 $6C
-    jr   AnimateTilesSlowSpeed                    ; $1C1C: $18 $06
+    jr   AnimateTilesSlowVelocity                 ; $1C1C: $18 $06
 
 AnimateWaterDungeonTilesGroup::
     ld   h, HIGH(AnimatedTiles) + $9              ; $1C1E: $26 $73
-    jr   AnimateTilesSlowSpeed                    ; $1C20: $18 $02
+    jr   AnimateTilesSlowVelocity                 ; $1C20: $18 $02
 
 AnimateSlowWaterfallTilesGroup::
     ld   h, HIGH(AnimatedTiles) + $0              ; $1C22: $26 $6A
 
-AnimateTilesSlowSpeed::
+AnimateTilesSlowVelocity::
```

Comment:
Hmmm… This is not related to an entity, but to an animation speed. IMO it should remain `AnimateTilesSlowSpeed`.